### PR TITLE
Add function to load a level from a TMX file

### PR DESCRIPTION
**Pull request by Ashthecool**
Implemented a tmx.level + added graphic bug.

Explanation to how it works is using "arcade.load_tilemap" I could load in tmx file into the game and put the different layers of tiles and then draw them.

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -29,6 +31,39 @@
 FIRE_KEY = arcade.key.SPACE
 
 
+def load_tmx(tmx_file, self):
```
Comment (tobiasbp) [inline my_game.py:34]: Add tile_scaling as an argument

File: my_game.py [hunk 2]
```diff
@@ -29,6 +31,39 @@
...
+        layer_names[3]: {"use_spatial_hash": True},
+        layer_names[4]: {"use_spatial_hash": True}
+    }
+
+    # Loads the tile-map, layer_options, and how big the tmx is.
+    tile_map = arcade.load_tilemap(
+        tmx_file, layer_options=layer_options, scaling=TILE_SCALING
+    )
+
+    self.dict_list = {
```
Comment (tobiasbp) [inline my_game.py:57]: No need for self (this is not a class/object)

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Added comments with things that need to be changed.

File: my_game.py [hunk 3]
```diff
@@ -29,6 +31,39 @@
...
+        "background": tile_map.sprite_lists[layer_names[0]],
+        "impassable": tile_map.sprite_lists[layer_names[1]],
+        "objects-impassable": tile_map.sprite_lists[layer_names[2]],
+        "objects-passable": tile_map.sprite_lists[layer_names[3]],
+        "pressure-plates": tile_map.sprite_lists[layer_names[4]]
+    }
+
+    return xx
```
[the remark this answers is not in the document]
Comment (Ashthecool) [reply my_game.py]: Done! :)

File: my_game.py (hunk 2 above)
Comment (Ashthecool) [reply my_game.py]: Changed! :]

File: my_game.py [hunk 4]
```diff
@@ -29,6 +31,40 @@
...
+        layer_names[0]: {"use_spatial_hash": True},
+        layer_names[1]: {"use_spatial_hash": True},
+        layer_names[2]: {"use_spatial_hash": True},
+        layer_names[3]: {"use_spatial_hash": True},
+        layer_names[4]: {"use_spatial_hash": True}
+    }
+
+    # Loads the tile-map, layer_options, and how big the tmx is.
+    tile_map = arcade.load_tilemap(
+        tmx_file, layer_options=layer_options, scaling=tile_scaling
```
Comment (tobiasbp) [inline my_game.py:54]: If it's called _scaling_ in _load_tilemap_ then use the same name in _load_tmx.

File: my_game.py [hunk 5]
```diff
@@ -29,6 +31,40 @@
 FIRE_KEY = arcade.key.SPACE
 
 
+def load_tmx(tmx_file, self, tile_scaling):
```
Comment (tobiasbp) [inline my_game.py:34]: Don't have the _self_ argument here. This is not in a class.

File: my_game.py [hunk 6]
```diff
@@ -29,6 +31,40 @@
...
+        layer_names[4]: {"use_spatial_hash": True}
+    }
+
+    # Loads the tile-map, layer_options, and how big the tmx is.
+    tile_map = arcade.load_tilemap(
+        tmx_file, layer_options=layer_options, scaling=tile_scaling
+    )
+
+    sprite_lists = {
+        "background": tile_map.sprite_lists[layer_names[0]],
```
Comment (tobiasbp) [inline my_game.py:58]: Don't type the name again:
```
layer_names[0]: tile_map.sprite_lists[layer_names[0]],
```

**Review by tobiasbp — CHANGES_REQUESTED** [review]
More changes needed. See comments.

**Pull request merged**